packages/styles/scss/components/breadcrumb/_breadcrumb.scss

```diff
@@ -0,0 +1,150 @@
+//
+// Copyright IBM Corp. 2018, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../breakpoint' as *;
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/component-reset';
+@use '../../utilities/convert';
+@use '../../utilities/skeleton' as *;
+
+@mixin breadcrumb {
+  .#{$prefix}--breadcrumb {
+    @include component-reset.reset;
```

Comment:
Same thing here with the `as *` import like the `convert`

Reply:
It seems like if we import `@use '../../type' as *` it's also pulling in a `reset`, so we may not even need this import 👍🏻